test/constructs/external-secrets.test.ts

```diff
@@ -0,0 +1,49 @@
+import '@aws-cdk/assert/jest';
+import * as eks from '@aws-cdk/aws-eks';
+import { Stack } from '@aws-cdk/core';
+import { ExternalSecrets } from '../../src/constructs/external-secrets';
+
+describe('external-secrets', () => {
+  test('namespace is "secrets"', () => {
+    const stack = new Stack();
+    const cluster = new eks.Cluster(stack, 'EKS', {
+      version: eks.KubernetesVersion.V1_18,
+    });
+    new ExternalSecrets(stack, 'ALB', {
```

Comment:
```suggest
    new ExternalSecrets(stack, 'ExternalSecrets', {
```

Reply:
thanks, now everyone sees my copy and paste skills :(